test/DynamoCoreTests/CodeBlockNodeTests.cs

```diff
@@ -1810,7 +1811,7 @@ public void ImportStatementInCodeBlock_DoesNotLoadAssemblyIntoProcess()
             string assemblyName = "FFITarget";
             UpdateCodeBlockNodeContent(codeBlockNode, $"import(\"{assemblyName}.dll\")");
 
-            var loadedAssemblies = AppDomain.CurrentDomain.GetAssemblies();
+            var loadedAssemblies = AssemblyLoadContext.All.SelectMany(context => context.Assemblies);
```

Comment:
whats the difference between these? Is CurrentDomain pointing to just the default ALC?